gslib/utils/iam_helper.py

```diff
@@ -52,11 +52,18 @@
     'allAuthenticatedUsers',
 ])
 
-# This is a convenience class to handle returned results from
+# These are convenience classes to handle returned results from
 # BindingStringToTuple. is_grant is a boolean specifying if the
-# bindings are to be granted or removed from a bucket / object,
-# and bindings is a list of BindingsValueListEntry instances.
+# bindings are to be granted or removed from a bucket / object.
+# bindings varies by type.
+# For BindingsTuple, it is a list of BindingsValueListEntry
+# For BindingsDictTuple it is a list of analogous dicts, e.g.:
+# {
+#   'members': ['member'],
+#   'role': 'role',
+# }
 BindingsTuple = namedtuple('BindingsTuple', ['is_grant', 'bindings'])
+BindingsDictTuple = namedtuple('BindingsDictTuple', ['is_grant', 'bindings'])
```

Comment:
The identical `namedtuple` is kind of a code smell

Reply:
Agreed, though "bindings" has different semantics here. I think ultimately it will help us keep message-dependent and non-message-dependent code separate.